src/drawer/components/WarningBox/WarningAccodion/WarningAccodion.style.ts

```diff
@@ -0,0 +1,42 @@
+import styled from 'styled-components';
+
+export const StyledAccordionContainer = styled.div`
+  width: 21.875rem;
+  height: fit-content;
+  border-radius: 0.75rem;
+  background: ${({ theme }) => theme.color.buttonDisabledBG};
+  padding-left: 1.12rem;
+  padding-top: 0.69rem;
+  padding-right: 1.25rem;
+  padding-bottom: 0.8rem;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  gap: 1.31rem;
+`;
+
+export const StyledAccordionHeaderContainer = styled.div`
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  justify-content: center;
+  gap: 5.45rem;
+`;
+
+export const StyledAccordionHeader = styled.div`
+  color: ${({ theme }) => theme.color.textPrimary};
+  ${({ theme }) => theme.typo.button3};
+`;
+
+export const StyledAccordionContent = styled.div<{ $isOpen: boolean }>`
+  width: 17.5rem;
+  color: ${({ theme }) => theme.color.textPrimary};
+  font-family: 'Apple SD Gothic Neo';
+  font-size: 0.65rem;
+  font-style: normal;
+  font-weight: 400;
+  line-height: 1.3;
+  margin-left: 0.5rem;
+
+  display: ${({ $isOpen }) => ($isOpen ? 'block' : 'none')};
+`;
```

Comment:
Accordion 열릴 때 애니메이션이 추가되면 좋을 것 같아요!

```suggestion
  margin-top: ${({ $isOpen }) => ($isOpen ? '1.31rem' : '0')};
  max-height: ${({ $isOpen }) => ($isOpen ? '500px' : '0')};
  overflow: hidden;
  transition:
    max-height 0.2s ease-out,
    margin-top 0.2s ease-out;
```

Reply:
오!!! 애니메이션 넣으면 좋을 거 같은데 현재 제 코드상으로 StyledAccordionContainer의 height가 fit-content로 되어 있어서 StyledAccordionContent의 내용물에 따라 높이가 달라지게 해놓은 상태인데..! 아코디언 박스들이 두개로 분기되어 있다면 애니메이션 처리가 가능할 것 같은데 이건 하나의 StyledAccordionContainer의 높이가 달라지는 거여서요!! 저희가 원하는 느낌의 애니메이션 넣을라믄 구조를 좀 바꿔야해서..

애니메이션 처리는.... 조금 후순위로 하면 어떨지... 조심스레 이야기해봅니다ㅎㅎ 호홓

Reply:
아하 그런 문제가 있었군요 그렇다면 애니메이션 적용은 추후에 하는걸로..